From 440cda8d2d88b5625289a42f759bd9c1829ba0f6 Mon Sep 17 00:00:00 2001
From: srskumar21 <srskumar21@gmail.com>
Date: Fri, 10 Apr 2020 16:09:24 +0530
Subject: [PATCH 3/8] Update test

---
 test | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test b/test
index 25bb6e3..3598feb 100644
--- a/test
+++ b/test
@@ -1 +1 @@
-This is to test, testing to merge.
+This is to test, testing to merge edited in dev
-- 
2.24.0.windows.2

