From a5eb4a9d6ef699840a309934cae27215eb1dc1eb Mon Sep 17 00:00:00 2001
From: srskumar21 <srskumar21@gmail.com>
Date: Fri, 10 Apr 2020 17:53:39 +0530
Subject: [PATCH 7/8] adding mergeTes file to Dev branch

---
 mergeTes | 1 +
 1 file changed, 1 insertion(+)
 create mode 100644 mergeTes

diff --git a/mergeTes b/mergeTes
new file mode 100644
index 0000000..a57ebd9
--- /dev/null
+++ b/mergeTes
@@ -0,0 +1 @@
+I am from mergetest file.
-- 
2.24.0.windows.2

